frameworks/C++/ffead-cpp/install_ffead-cpp-backends.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+cd $IROOT
+
+wget -q https://github.com/sumeetchhetri/ffead-cpp/archive/master.zip
```

Comment:
This and other places you pull from github must point to a specific release or hash, they can't be a branch or master. Your framework and dependencies can't be updated automatically without a pull request to make sure they pass travis.

Reply:
I have pinned the dependencies and the framework source code to specific commit id's now.